### DISCUSSION TITLE
Where can I see the code for the "Real Life Example"?

### DISCUSSION THREAD

**Discussion by oneoffaccount** [katoid/angular-grid-layout#12] [Q&A] [answered]
I would also like to see the code for the "Custom handles" example.

Thank you!  Great work with this library!

**Comment by llorenspujol** [comment, marked as answer]
Thank you:)

The code of 'real life example' is here: https://github.com/katoid/angular-grid-layout/tree/main/projects/demo-app/src/app/real-life-example
And the code of 'custom handles' here: https://github.com/katoid/angular-grid-layout/tree/main/projects/demo-app/src/app/custom-handles

I recommend you to **clone the repo** and start the application, is simple and you would have all the examples in your hand and you would be able to play and change them. The application that is shown with all 4 examples is under: angular-grid-layout/projects/demo-app
